app/src/main/java/org/simple/clinic/protocol/ProtocolRepository.kt

```diff
@@ -132,50 +132,50 @@ class ProtocolRepository @Inject constructor(
             )),
         ProtocolDrugAndDosages(
             drugName = "Chlorthalidone",
             drugs = listOf(
-                protocolDrug(UUID.fromString("d4e4cfc1-c30b-49a6-aede-718c7ef8ea36"), "Chlorthalidone", "12.5mg"),
-                protocolDrug(UUID.fromString("a2fb1e94-e76e-452b-8569-5ba06ac5a418"), "Chlorthalidone", "25mg")
+                protocolDrug(UUID.fromString("d4e4cfc1-c30b-49a6-aede-718c7ef8ea36"), "Chlorthalidone", "12.5mg", "331132"),
+                protocolDrug(UUID.fromString("a2fb1e94-e76e-452b-8569-5ba06ac5a418"), "Chlorthalidone", "25mg", "315655")
             )),
         ProtocolDrugAndDosages(
             drugName = "Losartan",
             drugs = listOf(
-                protocolDrug(UUID.fromString("320b50ed-d424-47c3-b1a7-4ef6a8e22c80"), "Losartan", "50mg"),
-                protocolDrug(UUID.fromString("06f2934b-5b02-4a6b-ae04-71eef8de02ef"), "Losartan", "100mg")
+                protocolDrug(UUID.fromString("320b50ed-d424-47c3-b1a7-4ef6a8e22c80"), "Losartan", "50mg", "979467"),
+                protocolDrug(UUID.fromString("06f2934b-5b02-4a6b-ae04-71eef8de02ef"), "Losartan", "100mg", "979463")
             )),
         ProtocolDrugAndDosages(
             drugName = "Atenolol",
             drugs = listOf(
-                protocolDrug(UUID.fromString("40d57e31-f209-4a48-86d6-6145031727e3"), "Atenolol", "25mg"),
-                protocolDrug(UUID.fromString("f40dd335-534d-471d-a77e-ed0622e0a521"), "Atenolol", "50mg")
+                protocolDrug(UUID.fromString("40d57e31-f209-4a48-86d6-6145031727e3"), "Atenolol", "25mg", "315437"),
+                protocolDrug(UUID.fromString("f40dd335-534d-471d-a77e-ed0622e0a521"), "Atenolol", "50mg", "315438")
             )),
         ProtocolDrugAndDosages(
             drugName = "Hydrochlorothiazide",
             drugs = listOf(
-                protocolDrug(UUID.fromString("875580d7-74d9-49e7-89f0-4d8d9df40021"), "Hydrochlorothiazide", "12.5mg"),
-                protocolDrug(UUID.fromString("3790002a-3093-4e42-bea2-3c4d893783ba"), "Hydrochlorothiazide", "25mg")
+                protocolDrug(UUID.fromString("875580d7-74d9-49e7-89f0-4d8d9df40021"), "Hydrochlorothiazide", "12.5mg", "316047"),
+                protocolDrug(UUID.fromString("3790002a-3093-4e42-bea2-3c4d893783ba"), "Hydrochlorothiazide", "25mg", "316049")
             )),
         ProtocolDrugAndDosages(
             drugName = "Aspirin",
             drugs = listOf(
-                protocolDrug(UUID.fromString("360df3c1-d00f-412a-9c71-bc58285f94e4"), "Aspirin", "75mg"),
-                protocolDrug(UUID.fromString("7a01d999-57d5-407d-a0a6-b74e28d7b3d0"), "Aspirin", "81mg")
+                protocolDrug(UUID.fromString("360df3c1-d00f-412a-9c71-bc58285f94e4"), "Aspirin", "75mg", "315429"),
+                protocolDrug(UUID.fromString("7a01d999-57d5-407d-a0a6-b74e28d7b3d0"), "Aspirin", "81mg", "315413")
```

Comment:
This is 150mg code and cannot find a 81mg Aspirin in the list. Can you take a look a this?

Reply:
Ah, good catch. Let me check with design folks.